dredd-rack.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.test_files = Dir["spec/**/*"]
 
   gem.add_dependency "capybara", "~> 2.4"
-  gem.add_dependency "rake", "~> 10.4"
+  gem.add_dependency "rake", ">= 10.4"
```

Comment:
We know that Rake 11 doesn't introduce breaking changes in it API (from the Dredd::Rack point of view - and thank you for your feedback!). However, if non-backward compatible change had to be introduced in the future, they are likely to be introduced on a major release.

Hi @tobiashm!

Using a [pessimistic version constrain](http://guides.rubygems.org/patterns/#pessimistic-version-constraint) (e.g. [`">= 10.4", "< 12"`](https://github.com/gonzalo-bulnes/dredd-rack/pull/31/commits/e86d10512f30007a397ba3a3f6e9987c8b812308#diff-0403706939d27e5c67edaa3ed1e8a62bR20)) makes sure such eventuality doesn't catch us by surprise.

(In particular, if someday Dredd::Rack behaves unexpectedly after explicitly upgrading its dependencies, the cause of the misbehaviour will be easy to find. If it starts misbehaving because of a completely un-related upgrade of Rake in your project, the origin of the problem can be much less obvious to spot.)